style.css

```diff
@@ -0,0 +1,50 @@
+#img-div {
+  width: 80%;
+  height: 80%;
+  margin: 8px 8px 8px 8px;
+  text-align: center;
+  padding: 20px 20px 20px 20px;
+  margin-left: 50px;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: auto;
+  margin-bottom: auto;
+  text-align: center;
+}
+
+main h1 {
+  text-align: center;
+  padding: 20px 20px 20px 20px;
+  background-color: lightgrey;
+  border: 2px solid black;
+}
+
+#tribute-info {
+  width: 100%;
+  height: 100%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 50px;
+  margin-bottom: 20px;
```

Comment:
please try to write this kind of styling code in their short terms to make your code dryer.